pkg/sdk/go/requests.go

```diff
@@ -19,3 +25,32 @@ type ConnectionIDs struct {
 	ChannelIDs []string `json:"channel_ids"`
 	ThingIDs   []string `json:"thing_ids"`
 }
+
+type issueKeyReq struct {
+	token    string
+	Type     uint32        `json:"type,omitempty"`
+	Duration time.Duration `json:"duration,omitempty"`
+}
+
+// It is not possible to issue Reset key using HTTP API.
```

Comment:
This is a confusing comment. This comment is supposed to describe what is implemented below, but looks like it does not.